### PR TITLE
Feature/footer update

### PR DESCRIPTION
**Pull request by JCourtney22**

### CODE REVIEW

**Review by JonHarrison — CHANGES_REQUESTED** [review]
-

File: src/components/AppFooter/style.css [hunk 1]
```diff
@@ -19,8 +19,6 @@
   padding: 10px;
 }
 
-.App-footer>div>.social-icon {
-  margin-left: 10px;
-  width: 16px;
-  height: 16px;
+.iconLinks{
+  color: #2b2828;
```
Comment (JonHarrison) [inline src/components/AppFooter/style.css]: would be better in a var

File: src/pages/Login/style.css [hunk 2]
```diff
@@ -1,3 +1,12 @@
 .App-login {
     width: 400px;
+    background-color: rgba(69, 156, 209, 0.5);
```
Comment (JonHarrison) [inline src/pages/Login/style.css:3]: would be better in a var

File: src/containers/Main/style.css [hunk 3]
```diff
@@ -2,4 +2,10 @@
     width: 100%;
     height: 100%;
     overflow: auto;
+    background-image: url('../../assets/images/Book\ pages.jpg');
```
Comment (JonHarrison) [inline src/containers/Main/style.css:5]: if you want the background on every page it's currently missing from the search page

File: src/containers/Main/style.css [hunk 4]
```diff
@@ -2,4 +2,10 @@
     width: 100%;
```
Comment (JonHarrison) [inline src/containers/Main/style.css:2]: non footer related changes would be better in another branch

File: src/containers/Main/style.css [hunk 5]
```diff
@@ -2,4 +2,10 @@
...
     height: 100%;
     overflow: auto;
+    background-image: url('../../assets/images/Book\ pages.jpg');
+    background-size:cover;
```
Comment (JonHarrison) [inline src/containers/Main/style.css:6]: non footer related changes would be better in another branch

File: src/pages/Login/index.js [hunk 6]
```diff
@@ -49,11 +49,12 @@ const Login = () => {
   };
 
   return (
+    <div className="loginDiv">
```
Comment (JonHarrison) [inline src/pages/Login/index.js:52]: non footer related changes would be better in another branch

File: src/pages/Login/style.css (hunk 2 above)
Comment (JonHarrison) [inline src/pages/Login/style.css:3]: non footer related changes would be better in another branch

**Review by JonHarrison — CHANGES_REQUESTED** [review]

File: src/components/AppFooter/style.css (hunk 1 above)
Comment (JCourtney22) [reply src/components/AppFooter/style.css]: Hmm thought I had - I will change it now.

File: src/pages/Login/style.css (hunk 2 above)
Comment (JCourtney22) [reply src/pages/Login/style.css:3]: If you still want me to do this, let me know.

File: src/containers/Main/style.css (hunk 3 above)
Comment (JCourtney22) [reply src/containers/Main/style.css:5]: Hmm yeh, the search page is a bit blank, do you want me to go ahead and change it still ?

File: src/containers/Main/style.css (hunk 5 above)
Comment (JCourtney22) [reply src/containers/Main/style.css:6]: ALthough it is a non-footer change, it was necessary to get the footer to had padding-top - would you rather me move this change to another branch still?

Comment (JCourtney22) [reply src/containers/Main/style.css:6]: Hmm maybe I didn't switch branches? I'll have to check this later if you still want me to move it to a different branch and re-push

File: src/containers/Main/style.css (hunk 4 above)
Comment (JCourtney22) [reply src/containers/Main/style.css:2]: I thought I had done this in another branch already - do you need me to redo this?

File: src/pages/Login/index.js (hunk 6 above)
Comment (JCourtney22) [reply src/pages/Login/index.js:52]: This is already in feature/login-page branch pull request, but as that hasn't been pulled across yet, that might be why it is appearing here as new.

File: src/pages/Login/style.css (hunk 2 above)
Comment (JCourtney22) [reply src/pages/Login/style.css:3]: This was also pushed into the feature/login-page branch, which hasn't been merged yet

**Review by JonHarrison — APPROVED** [review]
Approved with comments

**Pull request merged**